apps/tests/aws-runtime/test/tester.test.ts

```diff
@@ -484,6 +484,14 @@ function setupWS(executionId: string, ws: WebSocket) {
       reject(err);
     });
     ws.on("message", (data) => {
+      // test: invalid call, should not crash the server
```

Comment:
Has this been tested?

Reply:
yes

Reply:
Though it seems to be failing the runtime tests....